web/app/js/components/ErrorModal.jsx

```diff
@@ -10,59 +10,134 @@ export default class ErrorModal extends React.Component {
     resourceType: PropTypes.string.isRequired
   }
 
+  state = {
+    visible: false,
+    maxErrorLength: 500,
```

Comment:
It's a bit surprising to me that this is in the state, since it's never modified. It feel more like a constant to me.

Reply:
yeah, that's true, I should probably `const` it.